/dev/fd always contains the first 64 fd.

fd above 64 are not beeing closed.
when using dcheck, close returns errors on fd which are not open.

drop it, fallback on bruteforce implementation.

Index: base/process/launch_posix.cc
--- base/process/launch_posix.cc.orig
+++ base/process/launch_posix.cc
@@ -217,7 +217,7 @@ static const char kFDDir[] = "/dev/fd";
 #elif BUILDFLAG(IS_FREEBSD)
 static const char kFDDir[] = "/dev/fd";
 #elif BUILDFLAG(IS_OPENBSD)
-static const char kFDDir[] = "/dev/fd";
+static const char kFDDir[] = "";
 #elif BUILDFLAG(IS_ANDROID)
 static const char kFDDir[] = "/proc/self/fd";
 #endif
